src/components/Notification/Notification.stories.mdx

```diff
@@ -0,0 +1,180 @@
+    </NotificationContent>
+  </Notification>
+</Canvas>
+
+<Canvas>
+  <Notification notificationType={NotificationTypes.Warning}>
+    <NotificationHeading>Warning Notification</NotificationHeading>
+    <NotificationContent>
+      This is a "warning" Notification with a heading. Cras mattis consectetur
+      purus sit amet fermentum. Maecenas faucibus mollis interdum. Morbi leo
+      risus, porta ac consectetur ac, vestibulum at eros. Cum sociis natoque
+      penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vivamus
+      sagittis lacus vel augue laoreet rutrum faucibus dolor auctor.
+    </NotificationContent>
+  </Notification>
+</Canvas>
+
+### Without NotificationHeading
+
+<Canvas>
+  <Notification>
+    <NotificationContent>
+      This is a "standard" Notification without a heading. Cras mattis
+      consectetur purus sit amet fermentum. Maecenas faucibus mollis interdum.
+      Morbi leo risus, porta ac consectetur ac, vestibulum at eros. Cum sociis
+      natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+      Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor.
+    </NotificationContent>
+  </Notification>
+</Canvas>
+
+<Canvas>
+  <Notification notificationType={NotificationTypes.Announcement}>
+    <NotificationContent>
+      This is an "announcement" Notification without a heading. Cras mattis
+      consectetur purus sit amet fermentum. Maecenas faucibus mollis interdum.
+      Morbi leo risus, porta ac consectetur ac, vestibulum at eros. Cum sociis
+      natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+      Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor.
+    </NotificationContent>
+  </Notification>
+</Canvas>
+
+<Canvas>
+  <Notification notificationType={NotificationTypes.Warning}>
+    <NotificationContent>
+      This is a "warning" Notification without a heading. Cras mattis
+      consectetur purus sit amet fermentum. Maecenas faucibus mollis interdum.
+      Morbi leo risus, porta ac consectetur ac, vestibulum at eros. Cum sociis
+      natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
+      Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor.
+    </NotificationContent>
+  </Notification>
+</Canvas>
+
+### Custom Icon
+
+<Canvas>
+  <Notification
+    icon={<Icon name="check" size="large" color="section-research-secondary" />}
+  >
+    <NotificationHeading>Custom Icon</NotificationHeading>
+    <NotificationContent>
+      This is a Notification with a custom icon. Cras mattis consectetur purus
+      sit amet fermentum. Maecenas faucibus mollis interdum. Morbi leo risus,
+      porta ac consectetur ac, vestibulum at eros. Cum sociis natoque penatibus
+      et magnis dis parturient montes, nascetur ridiculus mus.
+    </NotificationContent>
+  </Notification>
+</Canvas>
+
+### Errors
```

Comment:
Would "Development Errors" be a better title? This would only really happen while actively working the DS component and is not a variant.

Reply:
Good suggesttion.  Done.